servicemodules/discord.py

```diff
@@ -144,45 +144,48 @@ async def on_message(message: discord.message.Message) -> None:
                         response_reasons=response.why,
                     )
 
-                top_response = responses.pop(0)
```

Comment:
When I was testing whether the fix worked, the top response was often GooseAI but it failed with the following error so I added this try-except so that if the top response fails, the next one can still be called

![Uploading Screenshot from 2022-09-25 19-14-06.png…]()